### PR TITLE
DBAASCLI-72 add gcloud to Jenkinsfile

### PR DESCRIPTION
**Pull request by nonemax**

### CODE REVIEW

**Comment by JNKPercona** [comment]
License check is ok.

**Comment by JNKPercona** [comment]
License check is ok.

**Comment by JNKPercona** [comment]
License check is ok.

**Comment by hors** [comment]
You need to send tests results to PR. Because for now you just run the tests and user can not see the results in PR.

**Comment by JNKPercona** [comment]
tests passed

**Comment by JNKPercona** [comment]
License check is ok.

**Comment by JNKPercona** [comment]
License check is ok.

**Comment by JNKPercona** [comment]
Tests passed

File: Jenkinsfile [hunk 1]
```diff
@@ -85,12 +175,26 @@ pipeline {
                                     -H "Authorization: token ${GITHUB_API_TOKEN}" \
                                     -d "{\\"body\\":\\"License check is ok. \\"}" \
                                     "https://api.github.com/repos/\$(echo $CHANGE_URL | cut -d '/' -f 4-5)/issues/${CHANGE_ID}/comments"
+                                    curl -v -X POST \
```
Comment (hors) [inline Jenkinsfile]:
```suggestion
                                curl -v -X POST \
```

File: Jenkinsfile [hunk 2]
```diff
@@ -35,6 +91,33 @@ pipeline {
...
+                    curl -s -L https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz \
+                        | sudo tar -C /usr/local/bin --strip-components 1 --wildcards -zxvpf - '*/oc'
+                    
+                    curl -s -L https://github.com/mitchellh/golicense/releases/latest/download/golicense_0.2.0_linux_x86_64.tar.gz \
+                        | sudo tar -C /usr/local/bin --wildcards -zxvpf -
+                  #  curl -s -L https://github.com/src-d/go-license-detector/releases/latest/download/license-detector.linux_amd64.gz \
+                  #      | gunzip | sudo tee /usr/local/bin/license-detector > /dev/null
+                    curl -s -L https://github.com/src-d/go-license-detector/releases/download/v3.0.2/license-detector.linux_amd64.gz \
+                        | gunzip | sudo tee /usr/local/bin/license-detector > /dev/null
+                    sudo chmod +x /usr/local/bin/license-detector
```
Comment (hors) [inline Jenkinsfile]: Why do we need to install it twice?

Also do you need openshift-origin-client-tools ?

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
Tests failed

**Comment by JNKPercona** [comment]
License check is ok.

**Comment by JNKPercona** [comment]
Tests passed

File: Jenkinsfile (hunk 1 above)
Comment (nonemax) [reply Jenkinsfile]: https://github.com/Percona-Lab/percona-dbaas-cli/pull/105/commits/1e1c15c33a8ef79ed2b740094940a28faefc1e1a

File: Jenkinsfile (hunk 2 above)
Comment (nonemax) [reply Jenkinsfile]: https://github.com/Percona-Lab/percona-dbaas-cli/pull/105/commits/1e1c15c33a8ef79ed2b740094940a28faefc1e1a

yes we need openshift-origin-client-tool

**Comment by JNKPercona** [comment]
License check is ok.

**Comment by JNKPercona** [comment]
Tests passed

**Review by hors — APPROVED** [review]

**Pull request merged**